### ISSUE TITLE
[Dribbblish] Spotify sidebar no longer shows icons

### ISSUE THREAD

**Issue by kaiyin2** [spicetify/spicetify-themes#750]
### 🔍 Have you checked Spicetify.app page for your issue?

- [X] I have checked the FAQ

### 🔍 Is there already an issue for your problem?

- [x] I have checked older issues, open and closed

### ℹ Environment / Computer Info

```Markdown
- Spotify version: 1.1.84.716 (latest version)
- Spicetify version: 2.9.8 (latest version)
```

### 📝 Description

Step 1: Launch Spotify on computer
Step 2: Spotify client begins to auto-update to the latest version
Step 3: Upgraded spicetify to the newest version and applied backup
Step 4: After spicetify launches Spotify, sidebar icons go missing.

### 📸 Screenshots

![bug](https://user-images.githubusercontent.com/67120455/166118901-88464af3-0a24-4c11-acd3-f19ad6907f5e.JPG)

**Comment by kaiyin2** [comment]
In the case that others have a similar issue, here is a **temporary workaround** for the problem:

1. Uninstall everything (**Spotify** and **spicetify**)
2. Install Spotify client version **1.1.78.765.g5ea20b00** from this [link](https://spotify.en.uptodown.com/windows/download/4431137).
3. Install the **latest version** (2.9.9) of **spicetify** from [here](https://github.com/spicetify/spicetify-cli/releases/tag/v2.9.9).
4. Apply necessary changes.

Looks like this for me:
![image](https://user-images.githubusercontent.com/67120455/170130464-3eb80c52-c96c-442e-9320-27b4b0aa1b14.png)

Hope to see a fix soon.

**Comment by afonsojramos** [comment]
This is related to https://github.com/spicetify/spicetify-cli/issues/1660, hopefully there will be a fix soon. I will leave this open for visibility.

**Comment by ps13g** [comment]
since 2.10.0 is out now will there be a fix for this theme because currently it does not work

**Comment by kaiyin2** [comment]
Just tried to update to the latest version of **spicetify (v. 2.10.1)** and the issue of the icons missing still persists.

I believe the workaround is still necessary. **See below:**

> In the case that others have a similar issue, here is a **temporary workaround** for the problem:
> 
> 1. Uninstall everything (**Spotify** and **spicetify**)
> 2. Install Spotify client version **1.1.78.765.g5ea20b00** from this [link](https://spotify.en.uptodown.com/windows/download/4431137).
> 3. Install the **latest version** (whatever that is) of **spicetify** from [here](https://github.com/spicetify/spicetify-cli/releases).
> 4. Apply necessary changes **(spicetify apply).**
> 
> Looks like this for me: ![image](https://user-images.githubusercontent.com/67120455/170130464-3eb80c52-c96c-442e-9320-27b4b0aa1b14.png)
> 
> Hope to see a fix soon.

**Comment by kyrie25** [comment]
Resolved via #793
